website/docs/r/oauth_client.html.markdown

```diff
@@ -51,7 +51,7 @@ The following arguments are supported:
   `https://api.github.com` or `https://ghe.example.com/api/v3`).
 * `http_url` - (Required) The homepage of your VCS provider (e.g.
   `https://github.com` or `https://ghe.example.com`).
-* `oauth_token` - (Required) The token string you were given by your VCS provider.
+* `oauth_token` - (Required) The personal token you were given by your VCS provider. eg, ghp_xxxxxxxxxxxxxxx
```

Comment:
How about we defer more to the API documentation for what this string represents and how to generate them, like this:

```suggestion
* `oauth_token` - (Required) The token string you were given by your VCS provider, e.g. `ghp_xxxxxxxxxxxxxxx` for a GitHub personal access token. For more information on how to generate this token string for your VCS provider, see the [Create an OAuth Client](https://www.terraform.io/docs/cloud/api/oauth-clients.html#create-an-oauth-client) documentation.
```